docs/src/content/docs/introduction/grants.mdx

```diff
@@ -47,48 +47,48 @@ The sequence of requests a client makes when setting up a payment can follow one
 
 #### Path 1 (most common)
 
-1. Request an `incoming-payment` grant from the payee's AS
-2. Send request to create an incoming payment resource to the payee's RS
-3. Request a `quote` grant from the payer's AS
-4. Send request to create a quote resource to the payer's RS
-5. Request an interactive `outgoing-payment` grant from the payer's AS
-6. Send request to create an outgoing payment resource to the payer's RS
+1. Request an `incoming-payment` grant from the authorization server of recipient's <Tooltip content="Account servicing entity" client: load><span>ASE</span></Tooltip>
```

Comment:
I've spent some time looking at this path and path 2. By spelling out authorization server and resource server, and by better indicating who the server belongs to, I think it makes the sentences too complex. In these specific instances, it's a case of something sounding good in theory but ending up not so good after execution. 

What do you think about this format for the paths?

1. Request an incoming-payment grant from the recipient-side authorization server.
2. Send request to create an incoming-payment resource to the recipient-side resource server.
3. Request a quote grant from the sender-side authorization server.
4. Etc...